/objdist:
  ros__parameters:
    z_max: 0.14
    z_min: -0.02
    distance: 0.8

